tests/functional/test_submitted_charts.py

```diff
@@ -246,23 +270,51 @@ def submission_tests_run_for_submitted_charts(secrets):
             r = github_api(
                 'post', f'repos/{secrets.test_repo}/git/refs', secrets.bot_token, json=data)
 
+            # Remove chart files from base branch
```

Comment:
This step is needed when the `test_repo:main` already contains the chart files and OWNERS file (this is true because the sandbox repo is in sync with prod repo). We can't create a PR with added files if the files are already in the branch.